Docs: updating script-run docs (#118)
terramate-io/terramate-docs: cli/reference/cmdline/script/script-run.md

# Script Run

::: info Note

> This is an upcoming experimental feature that is subject to change in the future. To use it now, you must enable the project config option `terramate.config.experiments = ["scripts"]`

Reviewer: ```suggestion
This is an experimental feature that might be subject to change in the future. To use it now, you must enable the project config option `terramate.config.experiments = ["scripts"]`
```

:::

> ### Overview

Reviewer: ```suggestion
## Overview
```

A [script](../../../orchestration/scripts.md#introduction) is a collection of different commands which are part of a workflow.
You can run a single `script run` command to run all the commands of the script block as one single executable unit without having to run them one by one.

> ### Usage

Reviewer: ```suggestion
## Usage
```
